client/blocks/product-purchase-features/product-purchase-features-list/jetpack-return-to-dashboard.jsx

```diff
@@ -4,14 +4,18 @@
 import React from 'react';
 import { localize } from 'i18n-calypso';
 import { get } from 'lodash';
+import unTrailingSlashIt from 'lib/route/untrailingslashit';
 
 /**
  * Internal dependencies
  */
 import PurchaseDetail from 'components/purchase-detail';
 
 export default localize( ( { selectedSite, translate } ) => {
-	const adminURL = get( selectedSite, 'options.admin_url', '' );
+	let adminURL = get( selectedSite, 'options.admin_url', '' );
+	if ( adminURL ) {
+		adminURL = unTrailingSlashIt( adminURL ) + '/admin.php?page=jetpack#/dashboard';
```

Comment:
Also, a very minor one, but I think the `#/dashboard` part can be removed from the URL. This would have the hidden bonus that If we decide to change the route name, or the routing structure in a future Jetpack version, this will assure that this link will work correctly.